Dockerfile

```diff
@@ -0,0 +1,44 @@
+# Build the pathfinder binary
+FROM rust:1.59-bullseye AS builder
+
+WORKDIR /usr/src/pathfinder
+COPY . .
+
+# Run pathfinder in a slimmer image
+RUN cargo build --release --bin pathfinder
+
+FROM debian:bullseye-slim AS runner
+
+RUN DEBIAN_FRONTEND=noninteractive \
+    && apt-get update \
+    && apt-get install -y ca-certificates tzdata libgmp-dev python3 python3-pip \
+    && rm -rf /var/lib/apt/lists/*
+
+COPY --from=builder /usr/src/pathfinder/target/release/pathfinder /usr/local/bin/pathfinder
+
+WORKDIR /usr/share/pathfinder
+COPY py py
+RUN python3 -m pip install --upgrade pip
+RUN python3 -m pip install -r py/requirements-dev.txt
+
+# Pathfinder expects a very specific directory structure for the
+# python environment and scripts
+# This docker image is not using virtual environments, so replicate
+# the directory structure with a shim python script.
+ENV VIRTUAL_ENV=/usr/share/pathfinder/py/venv
```

Comment:
@Mirko-von-Leipzig @CHr15F0x Is there a way we can change the Rust code so that it reads from the environments `python3` by default i.e. `#!/usr/bin/env python3`, or from configuration?

Reply:
I don't really understand these `venv` considerations or restrictions so I'm not really the best person to ask 😅

Reply:
I'll investigate, np :+1: .